src/js/media-controller.js

```diff
@@ -93,7 +93,16 @@ class MediaController extends MediaContainer {
 
     // Capture request events from internal controls
     const mediaUIEventHandlers = {
-      MEDIA_PLAY_REQUEST: () => this.media.play(),
+      MEDIA_PLAY_REQUEST: (e, media) => {
+        const streamType = Delegates[MediaUIAttributes.MEDIA_STREAM_TYPE](this);
+        const autoSeekToLive = this.getAttribute('autoseektolive');
```

Comment:
`autoseektolive=off` doesn't feel very htmly also increases inconsistency between attributes in media-chrome. That said, don't have any alternatives off the top of my head currently.

Reply:
yes for a boolean attribute we should try to not use `off`, `false` as value.
best to inverse it then.

`noseektolive`, `disableliveseek`, `noplaylive`

Reply:
Yeah naming this one was hard for me. Open to other ideas but I feel good enough about where it landed. It's also flexible incase we wanted to capture other options with it, like maybe one that tries to keep the playhead at the live edge even without unpausing.

> increases inconsistency between attributes in media-chrome

How so?

Reply:
`autohide="2"`
`gestures-disabled` why not `gestures="off"` then?
`nohotkeys` => better remove this one and add an `hotkeys="off"` option
`hotkeys="noarrowleft noarrowright"` 
`default-stream-type`
`audio`

I agree that the uniformness is starting to worsen between these handful of attributes.

we should decide 
- dashes vs no dashes
- how should a disabled attribute be. if it's `feature="off"` fine with me but best to keep it consistent

Reply:
I'd agree with @gkatsev and @luwes here. For an attribute currently named `autoseektolive`, I'm having a hard time imagining expanding values beyond a `boolean` (though that may just be a failure of imagination). As such, we might as well stick with both our convention and the common convention for boolean attributes in HTML: if the attribute is present (typically with a value of `""`), that implies `true`. Otherwise (aka it's not present), that implies `false`.

Reply:
Yeah I see the reason here, just trying to get this thing shipped already. 😆 Happy to give up on a multi-value approach if no one sees opportunity for expansion.

`<form>` at least has `novalidate`, so there's some precedence for no*, and I can get behind that. I think my favorite might be `noautoseektolive` (or dashed). `noseektolive` would be less verbose, but that kind of sounds like we're disabling the button from seeking to live too (which is why I added `auto` in the first place).

I can't think of any good option to reverse the relationship so it's not a negative. `manualseektolive`, `maintainliveplayheadposition`, `makeuserclicklivebuttontoseektolive`. :)